Sources/MapboxCoreNavigation/EHorizon/RoadObjectMatcher.swift

```diff
@@ -92,10 +92,16 @@ final public class RoadObjectMatcher {
 
      - parameter point: Point representing the object.
      - parameter identifier: Unique identifier of the object.
+     - parameter direction: Direction of the provided point, which is going to be matched.
      */
-    public func match(point: CLLocationCoordinate2D, identifier: RoadObjectIdentifier) {
-        let point = MatchableGeometry(id: identifier, coordinates: [point].map(CLLocation.init))
-        native.matchPoints(forPoints: [point], useOnlyPreloadedTiles: false)
+    public func match(point: CLLocationCoordinate2D, identifier: RoadObjectIdentifier, direction: CLLocationDirection? = nil) {
+        var heading: NSNumber? = nil
+        if let direction = direction {
```

Comment:
A negative `CLLocationDirection` value is documented to mean an invalid value, which MatchablePoint would probably expect to be represented by `nil` instead, so we should check for this case.